CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_salesforce v1.2.0
+This release includes the following updates:
+
+## Quickstart Fixes
+- Fixed casing syntax in `quickstart.yml` to match the default options in the Salesforce connector schema tab. Source tables are now in upper case, and snake casing is updated to camel casing. ([#61](https://github.com/fivetran/dbt_salesforce/pull/61))
+- Now Quickstart customers can now properly reference the source tables and enable or disable the models as they wish.
```

Comment:
Let's simplify this a bit to not cause confusion. Also, removed the second bullet as Quickstart users should not need to do anything and it should all be handled automatically. I don't want to call this out as they may think action is required, when it actually isn't.

```suggestion
- Fixed casing syntax in `quickstart.yml` to match the default options in the Salesforce connector schema tab. ([#61](https://github.com/fivetran/dbt_salesforce/pull/61))
```

Reply:
Updated.